### ISSUE TITLE
Add target prediction functionality.

### ISSUE THREAD

**Issue by mnowotka** [chembl/GLaDOS#312]
This should happen in three places of the web interface:

 - in compounds results there should be an option to get predicted targets
 - in targets results there should be an option to ger predicted compounds
 - in the (compounds x targets) matrix there should be a button to will empty cells with predicted values.

**Comment by Emma-Manners** [comment]
@nbosc I don't think this is relevant now that the target predictions are no longer integrated into the interface?

**Comment by nbosc** [comment]
Indeed, I developped the models a long time ago and for a while we were showing predictions on some compound report cards. However we could not streamline their training, there were some restrictions on the number of models we were using, and in the end it was decided (without asking my opinion) to drop them some time ago.
